source/main.cpp

```diff
@@ -31,7 +31,7 @@ MAIN_ACL(g_main_acl);
 UVISOR_SET_MODE_ACL(UVISOR_ENABLED, g_main_acl);
 
 DigitalOut led(MAIN_LED);
-Serial pc(USBTX, USBRX);
+Serial pc(STDIO_UART_TX, STDIO_UART_RX);
```

Comment:
Starting with mbed-drivers 0.12.0, you can use `get_stdio_serial` instead of using your own `Serial` object. For example: `get_stdio_serial().baud(115200);` See https://github.com/ARMmbed/mbed-drivers/commit/c0728ac3983f67b71d3410a07a7eb65b3d083774 for more details.